File: CMogore/fizzbuzz.java
/* 
QUESTION: Write a program that prints the numbers from 1 to 100. For multiples of 3, print Fizz; for
multiples of 5, print Buzz; and for numbers that are multiples of both 3 and 5, print FizzBuzz.

*/

public class fizzbuzz {
    public static void main(String[] args) {
        // Loop from 1 to 100
        for (int i = 1; i <= 100; i++) {
            // Check if the number is a multiple of both 3 and 5
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("FizzBuzz");
            }
            // Check if the number is a multiple of 3
            else if (i % 3 == 0) {
                System.out.println("Fizz");
            }
            // Check if the number is a multiple of 5
            else if (i % 5 == 0) {
                System.out.println("Buzz");
            }
            // If the number is neither a multiple of 3 nor 5, print the number
            else {
                System.out.println(i);
            }
        }
    }
}

File: mevinngugi/FizzBuzz.py
"""
Write a program that prints the numbers from 1 to 100. For multiples of 3, print Fizz; for
multiples of 5, print Buzz; and for numbers that are multiples of both 3 and 5, print FizzBuzz.
"""
def FizzBuzz():
    for x in range(1, 101):
        if x % 3 == 0 and x % 5 == 0:
            print (f"{x} % 3 and 5 = FizzBuzz")
        elif x % 3 == 0:
            print (f"{x} % 3 = Fizz")
        elif x % 5 == 0:
            print (f"{x} % 5 = Buzz")
